AirCasting/SessionViews/ShareSession/ShareSessionView.swift

```diff
@@ -6,13 +6,12 @@ import MessageUI
 import SwiftUI
 
 
-struct ShareView: View {
-    @Binding var showModal: Bool
+struct ShareSessionView<VM: ShareSessionViewModel>: View {
+    @ObservedObject var viewModel: VM
+    @Binding var showSharingModal: Bool
+    
+    // This email logic will be moved to view model in the next task
     @State var email: String = ""
-    @State var itemsForSharing: [String] = ["www.google.com"]
-    #warning("Implement working share sheet")
-    //To be sure that share sheet is working for now we are using "www.google.com" and then we will place some data there when making a logic to this
-    @State var showSheet = false
     @State var isShowingMailView = false
     @State var showingAlert = false
     @State var mailSendingResult: Result<MFMailComposeResult, Error>? = nil
```

Comment:
I know it was done by me, but I would like to ask you: Could you please move it to VM? If not I will do it in another task, let me know

Reply:
I left it for the task with sharing file via email and I intend to do it (as mentioned in the comment). Is that okay?

Reply:
Sure! Thank you a lot 🤖